server/index.jsx

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+
+const app = express();
+const bodyParser = require('body-parser');
+const db = require('../database/index.jsx');
+
+const port = process.env.PORT || 3003;
+
+
+app.use(express.static('./public'));
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json());
+
+app.engine('html', require('ejs').renderFile);
+
+app.get('/:id', (req, res) => {
+  res.render('../public/index.html');
+});
+
+app.get('/photos/:id', (req, res) => {
+  const requestExpId = parseInt(req.params.id, 10);
```

Comment:
What does the 10 as the second argument do?

Reply:
It's a very weird "missing radix parameter" adjustment.

Reply:
yep, you really need it!

Reply:
........math.